cmd/cluster/dynatrace/cluster.go

```diff
@@ -148,3 +153,36 @@ func GetDynatraceURLFromManagementCluster(clusterID string) (string, error) {
 	DTURL := fmt.Sprintf("%s://%s", DTApiURL.Scheme, DTApiURL.Host)
 	return DTURL, nil
 }
+
+func getConnection() (conn *sdk.Connection, error error) {
+	connection, err := utils.CreateConnection()
+	if err != nil {
+		return nil, err
+	}
+	defer connection.Close()
+
+	return connection, nil
+}
```

Comment:
**Question:** Does this work? It looks wrong to me: if I'm not mistaken, we would be closing the connection at the end of the `getConnection()` before returning it.